### DISCUSSION TITLE
Translate regular page slugs

### DISCUSSION THREAD

**Discussion by thiras** [vinissimus/next-translate#523] [Q&A] [answered]
Hello,

Is there any way to make regular pages (no get method on a page, just purely static pages) to use different slugs for each language?

With default locale `en` and a secondary language, `de` ;

```
example.com/about-us
example.com/de/uber-uns (instead of example.com/de/about-us)
```

**Comment by aralroca** [comment, marked as answer]
Yes, using [rewrites](https://nextjs.org/docs/api-reference/next.config.js/rewrites) + custom routing manifest + custom `<Link>` component:

https://github.com/vercel/next.js/discussions/18885
